packages/manager-react-components/src/components/templates/onboarding/onboarding.component.tsx

```diff
@@ -106,18 +102,8 @@ export const OnboardingLayout: React.FC<OnboardingLayoutProps> = ({
   return (
     <div className="flex flex-col mx-auto sm:px-10">
       {!hideHeadingSection && (
-        <section className="flex flex-col items-center">
-          {(img?.src || placeholderSrc) && (
-            <div className="flex justify-center pt-8 max-h-28">
-              <img
-                className="max-h-[150px]"
-                src={img?.src ?? placeholderSrc}
-                alt=""
-                width={img?.width}
-                height={img?.height}
-              />
-            </div>
-          )}
+        <section className="flex flex-col items-center mt-8">
+          <img className="max-h-28" src={placeholderSrc} {...img} />
```

Comment:
**nitpick(blocking):**`placeHolderSrc` is just an example image. We have to consider the `src` from `img` tag if it's present. This causes confusion if it display the `img.src` or `placeholderSrc`.

Reply:
Maybe we should concat img.className with the current className instead of overwriting it completely though

Reply:
It's a choice. 
- No do that allow to have more flexibility.
- Do that add behavior by default but can add adverse reaction on specific case.

Example : On MANAGER-14496. We have image with text title. So I want set `max-h-36`

It's easier it className is overwrite. And if we add a className. I think you can be responsible of all display rules.

After. May be export "DEFAULT_MAX_HEIGHT_CLASS" if we want add possibility to extends on parent component ?

I try to keep simple. That's how I did it. But not hesitate if you think it's better to do otherwise

Reply:
I agree with @chipp972 for concatenating the classname.

Reply:
Done. Not hesitate to say me if change is good for you ;)